locale/action_names.rb

```diff
@@ -1,5 +1,5 @@
 # Autogenerated!
-_("Action with sub plans")
-_("Import Puppet classes")
-_("Import facts")
 _("Remote action:")
+_("»Action with sub plans«")
+_("»Import Puppet classes«")
+_("»Import facts«")
```

Comment:
Latest tasks should be adding a newline at the end

Reply:
I have latest `develop`

Reply:
tasks don't have develop?